examples/arkode/CXX_serial/ark_advection_diffusion_reaction.cpp

```diff
@@ -72,8 +72,8 @@
  *      | IDX | D | R | Description                           |
  *      +-----+---+---+---------------------------------------+
  *      |  0  | E | E | fully explicit                        |
- *      |  1  | E | I | implicit reaction, explicit diffusion |
- *      |  2  | I | E | implicit diffusion, explicit reaction |
+ *      |  1  | E | I | implicit diffusion, explicit reaction |
+ *      |  2  | I | E | implicit reaction, explicit diffusion |
```

Comment:
```suggestion
 *      |  1  | I | E | implicit diffusion, explicit reaction |
 *      |  2  | E | I | implicit reaction, explicit diffusion |
```

Reply:
I adjusted the "I" and "E" to match the words to the right, but maybe the words to the right should be adjusted to match the "I" and "E"?